src/Microsoft.TestPlatform.Extensions.TrxLogger/ObjectModel/TestOutcome.cs

```diff
@@ -88,13 +88,20 @@ internal enum TestOutcome
     /// </summary>
     Pending,
 
+    // NOTE: OLD CODE WAS  Min = Error and Max = Pending
+    // This doesn't work well with the coversion to string for the reason explained here:
+    // https://learn.microsoft.com/en-us/dotnet/api/system.enum.tostring?view=net-7.0
```

Comment:
It's better not to force culture in URL

```suggestion
    // https://learn.microsoft.com/dotnet/api/system.enum.tostring?view=net-7.0
```